### DISCUSSION TITLE
READ IF BOOKMARKLETS AREN'T FUNCTIONING

### DISCUSSION THREAD

**Discussion by cinnamongold** [3kh0/ext-remover#700] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Hello everyone, if your bookmarklets aren't working, this is because _your device admin has disabled them._ Unfortunately, this program is almost completely useless to you if your admin has done this. For now, you should stick to simple blockers like [cosmicgeometry](https://cosmicgeometry.xyz). Thank you for reading this.

Also, for those of you feet and miles people, (americans) _DO NOT DOWNGRADE_, you WILL be punished and I promise, it will be severe. I mean _**expulsion**_. If nothing works, suck it up and do what you go to school to do.. or fuck it, just get on your phone and post to your snapchat story or something.

more info on [this post](https://github.com/3kh0/ext-remover/discussions/187)

**Comment by tedlesmemes2** [comment]
why will you get expelled
